### PR TITLE
Handcricket

### PR DESCRIPTION
**Pull request by kumarankm**
## Related Issue
  - Number Cricket Game(Hand Cricket)

Closes: #1403

### Describe the changes you've made
I have added hand cricket game with batting and bowling option as well as I have added button click for every balls

### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA

### Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] New and existing unit tests pass locally with my changes.

### Screenshots
Put any screenshot(s) of the project here.
h

### CODE REVIEW

**Comment by kumarankm** [comment]
@991rajat @Vishal-raj-1 @urvashi-code1255 Review it 😄

**Comment by kumarankm** [comment]
I have solved the conflicts. review it @991rajat

**Comment by kumarankm** [comment]
@Vishal-raj-1 @991rajat @harshgupta20   Kindly review please

File: script.js [hunk 1]
```diff
@@ -1546,6 +1546,12 @@ let projectData = [
   {
     projectName: "product Counter",
     projectImage: "assets/Images/product_counter.png",
+    projectUrl: "public/product_counter.html",
+  },
+  {
+    projectName: "Hand Cricket",
+    projectImage: "assets/GIFs/handcricket.gif",
+    projectUrl: "public/handcricket.html",
```
Comment (991rajat) [inline script.js:1554]: Fix this, Two Url are added.
Due to this last one is working, I,e on clicking product counter open.FIx it

**Review by 991rajat — COMMENTED** [review]
Checkout the changes.

**Comment by kumarankm** [comment]
> Checkout the changes.

@991rajat done 👍🏻

**Pull request merged**